### DISCUSSION TITLE
Fundo para Inclusão para a Python Brasil

### DISCUSSION THREAD

**Discussion by tiidadavena** [apyb/comunidade#40] [Geral]
Prezadas/os/es, apresentamos o Fundo para Inclusão:

A comunidade representada através da Organização da Python Brasil 2022, Associação Python Brasil, PyLadies Brasil e AfroPython, propuseram parte dos valores excedentes para a viabilização de diversidade e representatividade na maior conferência Python da América Latina.

Isto será feito através do redirecionamento dos recursos excedentes da Python Brasil do ano anterior para viabilizar a diversidade e representação destas minorias.

A gestão dos valores repassados serão feitas pelas PyLadies Brasil e AfroPython, que, juntamente com o Conselho e Diretoria, debateram, avaliaram e definiram os termos do fundo.

O fundo já tem aplicação prevista para a Python Brasil 2022 e esperamos contar uma representação mais ampla e diversa da comunidade Python brasileira!
 
Os termos do Fundo estão disponíveis no site da APyB: https://python.org.br/fundo-para-inclusão/

E o Formulário de Inscrição está disponível neste link (prazo até o dia 03/09/2022): https://2022.pythonbrasil.org.br/programa-auxilio-financeiro
![Post - Fundo para Inclusão(1)](https://user-images.githubusercontent.com/12106738/188182486-e37a6d88-5964-44ed-92f3-2f9a9f3fd360.png)

**Comment by cecivieira** [comment]
É muito feliz que a comunidade está madura o suficiente para ter esse tipo de ação. <3 <3

@tiidadavena @rougeth @muriloviana @anadulce A APyB não informou o montante inicial do Fundo, nem respondeu a [indagação na outra thread](https://github.com/apyb/comunidade/discussions/19#discussioncomment-3527922). Então, tendo em vista que o evento que deu origem ao fundo foi a Python Brasil 2021, considerando a [prestação de contas publicada deste site do evento](https://2021.pythonbrasil.org.br/prestacao-contas) e as [regras do Fundo](https://python.org.br/fundo-para-inclus%C3%A3o/), a @pyladies-brazil e @AfroPython estão considerando que o valor inicial do Fundo é de **R$ 102.160,72** .

Caso haja algum equivoco neste entendimento, solicitamos que seja explicitado com brevidade, pois isso impacta diretamente na quantidade de pessoas a serem beneficiadas pelo [Programa de Auxílio Financeiro para a Python Brasil 2022](https://github.com/pythonbrasil/pybr2022-bolsa-viagem).

----
cc @lipemorais

**Reply by muriloviana** [reply]
Olá @cecivieira, obrigado por nos marcar \o/

Verificamos na planilha financeira da APyB que os valores da Python Brasil 2021 estão em falta do lado da associação. Portanto, iremos realizar está revisão dos valores com o auxílio da prestação de contas da organização da PyBR 2021 e retornamos com todos os dados em nosso [repositório financeiro](https://github.com/apyb/financeiro), criado recentemente para tornarem públicas as movimentações da associação, além da confirmação do valor mencionado.

Finalizaremos esta revisão até o fim dessa semana, sem falta.

**Reply by muriloviana** [reply]
@cecivieira confirmado o valor do fundo: **R$102.160,72**
